pyro/distributions/transforms/neural_autoregressive.py

```diff
@@ -1,122 +1,20 @@
-# Copyright (c) 2017-2019 Uber Technologies, Inc.
+# Copyright Contributors to the Pyro project.
```

Comment:
Can you please preserve the old copyright below the new, as per our Uber council?